### DISCUSSION TITLE
Mailscanne / Cron hangs / email notification

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#17031] [Ignored/Incomplete]
Below happen without any notification to admin.  How to prevent this and / or how to notify admin every 10 min for instance? 

![image](https://user-images.githubusercontent.com/80203589/232756235-5fd73859-ca1d-4103-8cbf-3b4149823e42.png)

Best

**Comment by garryatbrighton** [comment]
It can be done with sql script and cron, but it seems to be internal app blocker, so maybe it can be resolved in core

**Comment by mariuszkrzaczkowski** [comment]
https://github.com/YetiForceCompany/YetiForceCRM/issues/13908

**Comment by garryatbrighton** [comment]
#mariuszkrzaczkowski thanks a lot for your valuable help :/

For all, 
create deletecrap.sql
with: use YOURDATABASE delete from vtiger_ossmails_logs where status = 1 limit 1;
add entry to crontab:
*/15 * * * * /usr/bin/mysql -uDATABASEUSER -pDATABASEPASSWORD < /var/www/html/PATHTOFILE/deletecrap.sql > /var/www/html/cache/logs/cronmysql.log 2>&1

Above will remove vtiger_ossmails_logs row with status 1 and unblock mailscanner so emails can keep being converted.
Solution not perfect as setting password in crontab is not recommended (unless you are the only root). Can use also other ways.

**Reply by mariuszkrzaczkowski** [reply]
rules for discussions/issues are to help everyone, not to clutter

**Comment by konadrian** [comment]
82f9c564eabeb5de552f02280ab92a1ee95807b2
Here you have commit with a fix. It's a simple change in logs.tpl file
